.vscode/settings.json

```diff
@@ -23,7 +24,5 @@
   },
   "python.languageServer": "Default",
   "python.analysis.typeCheckingMode": "basic",
-  "editor.rulers": [
-    88
-  ]
-}
\ No newline at end of file
+  "editor.rulers": [120]
```

Comment:
Was there any particular reason you changed this? It does'nt conform to our configured formatting rules.